tests/test_core.py

```diff
@@ -535,3 +536,12 @@ def test_ec_private_key(self):
         key = pem.parse(KEY_PEM_EC_PRIVATE)[0]
 
         assert isinstance(key, pem.ECPrivateKey)
+
+    def test_openshh_private_key(self):
+        """
+        Detects and loads private keys in the new OpenSSH private key format
+        (https://coolaj86.com/articles/the-openssh-private-key-format/)
+        """
+        key = pem.parse(KEY_PEM_OPENSSH)[0]
```

Comment:
It would be better to write `key, = ...` instead of `[0]` because that also ensures that the parser will only extract exactly one object.

Reply:
Done.